sql/5_0_1-to-5_0_2_upgrade.sql

```diff
@@ -669,3 +669,7 @@ CREATE TABLE `api_token` (
 ) ENGINE = InnoDB;
 #EndIf
 
+#IfMissingColumn users patient_menu_role
+ALTER TABLE `facility` ADD `mail_street` VARCHAR(30) NOT NULL AFTER `facility_taxonomy`, ADD `mail_city` VARCHAR(35) NOT NULL AFTER `mail_street`, ADD `mail_state` VARCHAR(2) NOT NULL AFTER `mail_city`, ADD `mail_zip` VARCHAR(10) NOT NULL AFTER `mail_state`;
```

Comment:
make newlines between each ADD and do not use AFTER

Reply:
Ok.